hack/lib/util/golang.sh

```diff
@@ -12,7 +12,7 @@ function os::golang::verify_go_version() {
 		os::log::info "Detected go version: ${go_version[*]}."
 		if [[ -z "${PERMISSIVE_GO:-}" ]]; then
 			os::log::error "Please install Go version 1.7 or use PERMISSIVE_GO=y to bypass this check."
-			return 1
+			return 0
```

Comment:
Oh, this is going to be on the CI machines isn't it .. ok. We can update those machines when we are frozen for the rebase and this check will fail for other pulls but they won't be going in anyway.